src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.company.metrics.service;
+
+import org.springframework.beans.factory.annotation.Lookup;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.model.ApiResponse;
+
+@Service
+public class CompanyMetricsApiService {
+
+    /**
+     * Invoke Insolvency API.
+     */
+    public ApiResponse<?> invokeInsolvencyApi() {
```

Comment:
Shouldn't this method be called invokeCompanyMetricsApi? And also the related comment...

Reply:
good spot. incorporated the change in file.

Reply:
Thanks